tests/cbtest/gtest/test_uzfs.cc

```diff
@@ -816,28 +1033,92 @@ TEST(uZFS, TestRebuild) {
 
 	zvol_rebuild_step_size = (1024ULL * 1024ULL * 1024ULL) / 2 + 1000;
 	/* thread that helps rebuilding exits abruptly just after connects */
-	execute_rebuild_test_case("rebuild abrupt", 1, ZVOL_REBUILDING_IN_PROGRESS);
+	execute_rebuild_test_case("rebuild abrupt", 1, ZVOL_REBUILDING_IN_PROGRESS, B_FALSE, B_FALSE);
```

Comment:
I think we don't need this function anymore.

Reply:
just to clarify what I meant .. we don't need the enclosing function. execute_rebuild_test_case is obviously still needed.